utils/antlr/ClickHouseParser.g4

```diff
@@ -505,7 +505,16 @@ keyword
     | UNBOUNDED | UNION | UPDATE | USE | USING | UUID | VALUES | VIEW | VOLUME | WATCH | WHEN | WHERE | WINDOW | WITH
     ;
 keywordForAlias
-    : DATE | FIRST | ID | KEY
+    : AFTER | ALIAS | ALTER | ASCENDING | AST | ASYNC | ATTACH | BOTH | BY | CASE | CAST | CHECK | CLEAR | CLUSTER | CODEC | COLLATE
+    | COLUMN | COMMENT | CONSTRAINT | CREATE | CUBE | CURRENT | DATABASE | DATABASES | DATE | DEDUPLICATE | DEFAULT | DELAY | DELETE
+    | DESCRIBE | DESC | DESCENDING | DETACH | DICTIONARIES | DICTIONARY | DISK | DISTRIBUTED | DROP | ELSE | END | ENGINE | EVENTS
+    | EXISTS | EXPLAIN | EXPRESSION | EXTRACT | FETCHES | FIRST | FLUSH | FOR | FOLLOWING | FREEZE | FUNCTION | GRANULARITY | HIERARCHICAL
+    | ID | IF | INDEX | INJECTIVE | INSERT | INTERVAL | IS_OBJECT_ID | KEY | KILL | LAST | LAYOUT | LEADING | LIFETIME | LIVE | LOCAL
```

Comment:
based on comment below it is bad
```
// |interval| can't be an alias, otherwise 'INTERVAL 1 SOMETHING' becomes ambiguous.
```
And I have no good way to check that this list is good. For example CASE.

Reply:
cc @alexey-milovidov @qoega 
Sorry, I received the notification from Github a little late so I didn't reply on time.
The behaviour can be verified by comparing the ANTLR and the Clickhouse official parser.
Since the PR has been merged, I will do the job soon.